zkvm/docs/zkvm-spec.md

```diff
@@ -486,47 +525,24 @@ Payload of a [contract](#contract-type) may contain arbitrary [types](#types),
 but in the [output](#output-structure) only the [portable types](#portable-types) are allowed.
 
 
-### Input structure
-
-Input structure represents an unspent output (UTXO) from a previous transaction.
-
-Input is serialized as [output](#output-structure) with an extra 32 bytes containing
-the output’s [transaction ID](#transaction-id).
-
-```
-       Input  =  PreviousTxID || PreviousOutput
-PreviousTxID  =  <32 bytes>
-
-```
-
-### UTXO
-
-UTXO is an _unspent transaction output_ identified by a 32-byte hash computed using [transcript](#transcript):
-
-```
-T = Transcript("ZkVM.utxo")
-T.commit("txid", previous_txid)
-T.commit("output", previous_output)
-utxo = T.challenge_bytes("id")
-```
-
-In the above, `previous_txid` is the [transaction ID](#transaction-id) of the transaction where the output was created,
-and `previous_output` is the serialized [output](#output-structure).
-
-
 ### Output structure
 
 Output represents a _snapshot_ of a [contract](#contract-type)
```

Comment:
Putting snapshot in italics sort of implies it's a technical term whose definition is nearby. Perhaps it should not be in italics and you can get away with saying something like ..."a snapshot of a contract's state at the moment it executed an `output` instruction."

Reply:
yeah, makes sense

Reply:
changed to

```
Output is a serialized [contract](#contract-type):
```